sdk/ml/azure-ai-ml/azure/ai/ml/operations/_data_operations.py

```diff
@@ -347,6 +353,63 @@ def create_or_update(self, data: Data) -> Data:
                     )
             raise ex
 
+    @experimental
+    def import_data(self, data_import: DataImport) -> Job:
+        """Returns the data import job that is creating the data asset.
+
+        :param data_import: DataImport object.
+        :type data_import: azure.ai.ml.entities.DataImport
+        :return: data import job object.
+        :rtype: ~azure.ai.ml.entities.Job
+        """
+
+        experiment_name = "data_import_" + data_import.name
+        data_import.type = AssetTypes.MLTABLE if isinstance(data_import.source, Database) else AssetTypes.URI_FOLDER
+        if "{name}" not in data_import.path:
+            data_import.path = data_import.path.rstrip("/") + "/{name}"
+        import_job = import_data_func(
+            description=data_import.description or experiment_name,
+            display_name=experiment_name,
+            experiment_name=experiment_name,
+            compute="serverless",
+            source=data_import.source,
+            outputs={
+                "sink": Output(
+                    type=data_import.type, path=data_import.path, name=data_import.name, version=data_import.version
+                )
+            },
+        )
+        import_pipeline = PipelineJob(
+            description=data_import.description or experiment_name,
+            tags=data_import.tags,
+            display_name=experiment_name,
+            experiment_name=experiment_name,
+            properties=data_import.properties or {},
+            settings=PipelineJobSettings(force_rerun=True),
+            jobs={experiment_name: import_job},
+        )
+        import_pipeline.properties["azureml.materializationAssetName"] = data_import.name
+        return self._job_operation.create_or_update(job=import_pipeline, skip_validation=True)
+
+    @experimental
+    def show_materialization_status(
```

Comment:
show_materialization_status the name feels strange for a function return a list of jobs

Reply:
looks like this is defined by the spec: https://github.com/Azure/azureml_run_specification/blob/master/specs/python-sdk/import-data-python.md

Reply:
I'm glad I'm not the only who feels this name is too long. :)